x-pack/test/common/services/spaces.ts

```diff
@@ -24,27 +24,27 @@ export function SpacesServiceProvider({ getService }: FtrProviderContext) {
 
   return new (class SpacesService {
     public async create(space: any) {
-      log.debug('creating space');
+      log.debug(`creating space ${space.name}`);
       const { data, status, statusText } = await axios.post('/api/spaces/space', space);
 
       if (status !== 200) {
         throw new Error(
           `Expected status code of 200, received ${status} ${statusText}: ${util.inspect(data)}`
         );
       }
-      log.debug('created space');
+      log.debug(`created space ${space}`);
     }
 
     public async delete(spaceId: string) {
-      log.debug(`deleting space: ${spaceId}`);
+      log.debug(`deleting space id: ${spaceId}`);
       const { data, status, statusText } = await axios.delete(`/api/spaces/space/${spaceId}`);
 
       if (status !== 204) {
-        throw new Error(
+        log.debug(
```

Comment:
I just want to call this out for reviewers.  The issue here is that some tests are still unloading an es_archive which contains spaces.  This method was failing if trying to delete a space which has already been removed by unloading the es_archive.  It appears that this is primarily used as a tearDown step and not any tests.

Reply:
Yeah my eyebrow raised on this one at first, but I concur.  Good call.